components/board/genericlinux/hw_pwm.go

```diff
@@ -99,7 +99,7 @@ func (pwm *pwmDevice) unexport() error {
 	// On boards like the Odroid C4, there is a race condition in the kernel where, if you unexport
 	// the pin too quickly after changing something else about it (e.g., disabling it), the whole
 	// PWM system gets corrupted. Sleep for a small amount of time to avoid this.
-	time.Sleep(time.Millisecond)
+	time.Sleep(10 * time.Millisecond)
```

Comment:
started seeing the crashing issue again when i was testing the pins, only happens if you are switching between gpio and pwm quickly so shouldn't be a very common issue. Increased the sleep to 10ms seemed to resolve it.